### DISCUSSION TITLE
Custom Quick Response voice ids

### DISCUSSION THREAD

**Discussion by rydercowan** [fuatakgun/eufy_security#1205] [Q&A] [answered]
How can I find the voice ids for the custom quick responses created in Eufy app, so I can use them in automation?

**Comment by fuatakgun** [comment, marked as answer]
You should see them under Debug Device sensor attributes.

![Screenshot_2024-09-17-17-04-56-605_io.homeassistant.companion.android-edit.jpg](https://github.com/user-attachments/assets/75c7ec39-6634-4000-b693-c06ca6cb4790)

**Reply by rydercowan** [reply]
Thank you. I'm a novice, and have no idea where to find 'Debug device sensor attributes'! Could you point me in the right direction, please?

**Reply by rydercowan** [reply]
I eventually found it!. Thanks again for your help.
